src/Translator.php

```diff
@@ -331,9 +325,9 @@ public function pgettext(string $msgctxt, string $msgid): string
      */
     public function npgettext(string $msgctxt, string $msgid, string $msgidPlural, int $number): string
     {
-        $key = implode(chr(4), [$msgctxt, $msgid]);
+        $key = $msgctxt . "\u{4}" . $msgid;
         $ret = $this->ngettext($key, $msgidPlural, $number);
-        if (strpos($ret, chr(4)) !== false) {
+        if (str_contains($ret, "\u{4}")) {
```

Comment:
Here you used `str_contains` function but for the `pgettext` method you used `$ret === $key`. What's the reasoning for it?

Reply:
I could not verify whether the returned value is the exact same or some combination of input string. I can use `=== $key` if you say it's ok. I just wasn't sure it's safe to do so.